### ISSUE TITLE
Generate Dataset fails with larger dimension

### ISSUE THREAD

**Issue by len0rd** [BYU-AUVSI/imaging#18]
generate_dataset.jsx is able to generate images for `DIM_ = 100` just fine. When I crank it up to `DIM_ = 200`, I get an error from line 77 of drawShape, when trying to draw a quarterCircle.

@BrandonMcBride4 any ideas on this? The radii you're giving quarter and semi-circle make sense to me so im not sure. I'm also having a hard time fully understanding the drawShape function

**Comment by BrandonMcBride4** [comment]
Yeah the circles aren't completely circles, they are just a polygon with lots of sides
